hv-rhel6.x/hv/storvsc_drv.c

```diff
@@ -2289,10 +2291,19 @@ static int storvsc_probe(struct hv_device *device,
 		host->nr_hw_queues = stor_device->num_sc + 1;
 #endif
 
+	/*
+	 * Set the error handler work queue.
+	 */
+	host_dev->handle_error_wq =
+			alloc_ordered_workqueue("storvsc_error_wq_%d",
```

Comment:
In 2.6.32-754.el6.x86_64, alloc_ordered_workqueue takes only 2 args.
/usr/src/kernels/2.6.32-754.el6.x86_64/include/linux/workqueue.h:#define alloc_ordered_workqueue(name, flags) create_singlethread_workqueue(name)